Gemfile

```diff
@@ -60,3 +62,8 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+group :production do
+  gem 'rails_12factor'
+
+end
```

Comment:
@sansuaza the file should end within a blank space. Please fix it across your project!

Reply:
done!